src/handlers/input.rs

```diff
@@ -26,15 +27,15 @@ pub fn handler(key: Key, app: &mut App) {
         }
         Key::Left => {
             if !app.input.is_empty() && app.input_idx > 0 {
-                let last_c = app.input.chars().nth(app.input_idx - 1).unwrap();
+                let last_c = app.input[app.input_idx - 1];
```

Comment:
Could there ever be a situation where `app.input_idx - 1` goes out of bounds?

Perhaps it's safer to do `app.input.get(app.input_idx - 1)` and then handle the `Option`?

Reply:
it is guarded by the condition `app.index_idx > 0` so it's safe